internal/repo/repo.go

```diff
@@ -34,7 +34,9 @@ func CloneRepo(cloneURL, branch, token string) (*git.Repository, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), time.Second*120)
 	defer cancel()
 
-	repo, err := git.PlainCloneContext(ctx, "./out_repo", false, &git.CloneOptions{
+	dir := fmt.Sprintf("%s/%s-%d", os.TempDir(), branch, time.Now().Unix())
```

Comment:
On unix this look like:

`/tmp/<branch>-<time>`

The name of the directory does not matter, because its part of the handle returned by the function. (`repo` variable)